.github/workflows/daily_ete_test.yml

```diff
@@ -68,64 +93,89 @@ jobs:
         run: |
           python3 -m pip list
           lmdeploy check_env
+          rm -rf allure-results
       - name: Test lmdeploy - quantization w4a16
         continue-on-error: true
+        if: contains(fromJSON(github.event.inputs.backend), 'turbomind') && contains(fromJSON(github.event.inputs.model), 'quantization')
```

Comment:
you have to consider case when the job is triggered as schedule and all values of ``github.event.inputs`` is empty.